flow/Trace.cpp

```diff
@@ -41,6 +43,19 @@
 #undef min
 #endif
 
+Reference<ITraceLogFormatter> createLogFormatter() {
+	auto f = FLOW_KNOBS->TRACE_FORMAT;
+	std::transform(f.begin(), f.end(), f.begin(), ::tolower);
+	if (f == "json") {
+		return Reference<ITraceLogFormatter>(new JsonTraceLogFormatter());
+	} else if (f == "xml") {
+		return Reference<ITraceLogFormatter>(new XmlTraceLogFormatter());
+	} else {
+		ASSERT(false);
+		return Reference<ITraceLogFormatter>(new XmlTraceLogFormatter());
```

Comment:
This return statement should be deleted.

Reply:
Apparently you can add this change in a comment - not sure how...
```suggestion
-		return Reference<ITraceLogFormatter>(new XmlTraceLogFormatter());
```